packages/account/src/Sections/Verification/ProofOfOwnership/index.js

```diff
@@ -0,0 +1 @@
+export default from './proof-of-ownership.jsx';
```

Comment:
I think eslint disallows this. I have eslint errors when using it like this. check it and if you have an error, try import and then export it.

Reply:
@iman-fs, This error only pops up for tsx files, but I'll change it since we are moving to tsx anyway.